src/ralph/models/converter.py

```diff
@@ -0,0 +1,82 @@
+"""Base converter definition"""
+
+from abc import ABC, abstractmethod
+from uuid import UUID, uuid5
+
+from pydantic import BaseModel
+
+from ralph.defaults import MODEL_PATH_SEPARATOR
+from ralph.exceptions import ConversionException
+from ralph.utils import get_dict_value_from_path, set_dict_value_from_path
+
+
+class BaseConverter(ABC):
+    """Converter Base Class.
+
+    Converters define a conversion set to convert from one event type to another.
+    """
+
+    __model__: BaseModel
+
+    def __init__(self, uuid_namespace: UUID):
+        """Initializes BaseConverter."""
+
+        self.uuid_namespace = uuid_namespace
+        self.field_array = []
+        self.prepare_converter(self.field_array, self.get_conversion_set())
```

Comment:
I'm hesitating about this, the preparation step is used to split the source and destination paths into lists once the object is initialized. (We could keep the same object to convert multiple events in a row).
I could move this part to the convert function (but then it will be re-executed for each event). WDYT ?

Reply:
I think it will only have a minor performance impact but you are right. You can leave it as is for now.